### PR TITLE
www.businesstoday.in

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/115114

<details><summary>Screenshot:</summary>

Снимок экрана 2022-04-12 в 14 19 46

Снимок экрана 2022-04-12 в 14 23 08

Снимок экрана 2022-04-12 в 14 23 27

Снимок экрана 2022-04-12 в 14 23 35

</details><br/>

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -122,6 +122,8 @@ theoryandpractice.ru##a[href^="https://form.typeform.com/"]
 !
 softbank.jp##.mbox-banner
 shingakunet.com##.l-footerFix
+businesstoday.in##.bottomSliderSwipe
+businesstoday.in##.top_stories_breaking
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt:126]: `businesstoday.in##.top_stories_breaking`
Is it reqlly annoyung? I see it on the main page at the top only.

**Review by Alex-302 — COMMENTED** [review]
I see newsletter widget under articles, can be blocked
`businesstoday.in##.newsltter-iframe`

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (jellizaveta) [reply AnnoyancesFilter/sections/popups.txt:126]: `businesstoday.in##.newsltter-iframe` already blocked by `Fanboy's Annoyances`

Comment (zloyden) [reply AnnoyancesFilter/sections/popups.txt]: It's a third party filter, you should block it in AdGuard Annoyances.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**